include/libnuraft/raft_params.hxx

```diff
@@ -74,6 +74,7 @@ struct raft_params {
         , log_sync_stop_gap_(99999)
         , snapshot_distance_(0)
         , snapshot_block_size_(0)
+        , snapshot_sync_ctx_timeout_(2500)
```

Comment:
Put `0` by default, and if zero, keep the current behavior (`heart_beat_interval_ * response_limit_`). There are lots of users with different heartbeats, and such a newly introduced parameter with a fixed default value will suddenly break their systems.